modules/templates/currency-elements.xml

```diff
@@ -0,0 +1,104 @@
+<!--
+  ~ Copyright (C) 2023 Xibo Signage Ltd
+  ~
+  ~ Xibo - Digital Signage - https://xibosignage.com
+  ~
+  ~ This file is part of Xibo.
+  ~
+  ~ Xibo is free software: you can redistribute it and/or modify
+  ~ it under the terms of the GNU Affero General Public License as published by
+  ~ the Free Software Foundation, either version 3 of the License, or
+  ~ any later version.
+  ~
+  ~ Xibo is distributed in the hope that it will be useful,
+  ~ but WITHOUT ANY WARRANTY; without even the implied warranty of
+  ~ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  ~ GNU Affero General Public License for more details.
+  ~
+  ~ You should have received a copy of the GNU Affero General Public License
+  ~ along with Xibo.  If not, see <http://www.gnu.org/licenses/>.
+  -->
+<templates>
+    <template>
+        <id>currency_name</id>
+        <extends override="text" with="data.NameShort">text</extends>
+        <title>Currency Name</title>
+        <type>element</type>
+        <dataType>currency</dataType>
+        <canRotate>true</canRotate>
+        <thumbnail>currency-name</thumbnail>
+        <startWidth>650</startWidth>
+        <startHeight>250</startHeight>
+        <assets>
+            <asset id="currency-name" type="path" cmsOnly="true" mimeType="image/png" path="/modules/assets/template-thumbnails/currency/elements/currency-name.png" />
+        </assets>
+    </template>
+    <template>
+        <id>currency_rawLastTradePriceOnly</id>
+        <extends override="text" with="data.RawLastTradePriceOnly">text</extends>
+        <title>Last Trade Price</title>
+        <type>element</type>
+        <dataType>currency</dataType>
+        <canRotate>true</canRotate>
+        <thumbnail>currency-last-trade-price</thumbnail>
+        <startWidth>650</startWidth>
+        <startHeight>250</startHeight>
+        <assets>
+            <asset id="currency-last-trade-price" type="path" cmsOnly="true" mimeType="image/png" path="/modules/assets/template-thumbnails/currency/elements/currency-last-trade-price.png" />
+        </assets>
+    </template>
+    <template>
+        <id>currency_changePercentage</id>
+        <extends override="text" with="data.ChangePercentage">text</extends>
+        <title>Change Percentage</title>
+        <type>element</type>
+        <dataType>currency</dataType>
+        <canRotate>true</canRotate>
+        <thumbnail>currency-change-percentage</thumbnail>
+        <startWidth>650</startWidth>
+        <startHeight>250</startHeight>
+        <onElementParseData><![CDATA[
```

Comment:
This is where `onElementParseData` is currently used since we have **%** string for **Change Percentage** property of the currency.